src/InputHandler.js

```diff
@@ -0,0 +1,29 @@
+import {
+  getPointer,
+  initInput,
+  initPointer,
+  onInput,
+  onKey,
+  pointerPressed,
+} from "kontra";
+
+export default class InputHandler {
+  constructor(selected) {
+    initPointer();
+    initInput();
+
+    this.selected = selected;
+    onInput("down", () => {
+      if (pointerPressed("left")) {
+        const pointer = getPointer();
+        this.selected.target = { x: pointer.x - 10, y: pointer.y - 20 };
```

Comment:
I should probably utilize the height and width of the sprite here to do the subtraction.

Reply:
Yea, or potentially handle that within the selected object itself.

Each character would have a potential "target":
- Ranged DPS: enemy, wouldn't have to move if within range, can just shoot em
- Tank: enemy, would have to move to within range
- Healer: friendly, probably wouldn't have to move, just heal
- Enemies: "player character"